tasks/gotify.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Gotify Directory
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ gotify_data_directory }}/app/data"
```

Comment:
It feels a little clunky adding `/app/data` to the end of the path, as data is then stored in `{{ docker_home }}/gotify/config/app/data`

Reply:
Intentional. (Like the other reference.) I like to make the dirs match in the config directory to the container. Personal preference only and I'm not highly committed to it.

Reply:
BTW: NO config dir.
I'm going to start getting away from the template docker_data/app/config default and go with docker_data/app as the "config"/"data" dir. I really like matching the container's absolute path to a path under docker_data/app - visually much easier to see what's going on and debug when needed.